src/NDItk.hpp

```diff
@@ -3,5 +3,6 @@
 #include "NDItk/multigroup/FluxWeights.hpp"
 #include "NDItk/multigroup/CrossSection.hpp"
 #include "NDItk/multigroup/ReactionCrossSections.hpp"
+#include "NDItk/multigroup/FissionEnergyRelease.hpp"
```

Comment:
Here it's FissionEnergyRelease while below it's AverageFissionEnergyRelease. Typo?

Reply:
Yes, that's the old name so this is wrong. I changed the name of the class to AverageXXXX in case we wanted to add multigroup energy release components one day.